multi-server-install-main.adoc

```diff
@@ -297,14 +307,15 @@ Select from menu, or press 'a' to apply config (? - help)
 ....
 Common configuration
 
-   1) Hostname:                                ldap-1.example.com
-   2) Ldap master host:                        ldap-1.example.com
+   1) Hostname:                                ldap1.example.com
```

Comment:
there is a bit of mixing of `ldap-1` and `ldap1`?  I think the convention including the `-` makes most sense as other server types use similar naming.